index.html

```diff
@@ -96,14 +133,21 @@ <h1>Multi-Post
                </ul>
                <div class="external_link">
                    <a href="#">See Project</a>
-               </div>
+               
+              </div>
               </footer>
   
             
           </section>
           <section id="p3">
             <div id="introd_3">
+              
+                <div class="row">
+                  <div class="column">
+      
```

Comment:
As requested by the previous reviewer, kindly update the margin on your projects section. Currently, there is too much space between the card and the body border 😉.

| Figma design  | Your design |
| ------------- | ------------- |
| <img src="https://user-images.githubusercontent.com/95243996/201197659-279838ca-55cb-4b8b-949f-423e2aecad5b.png"/>  | <img src="https://user-images.githubusercontent.com/95243996/201197817-e5919336-d89e-4470-bc9d-03d3970dd2d3.png" />  |

> Keep in mind that this project's goal is to improve your ability to follow design guidelines (imagine Microverse as a client requesting a website) 😉.